src/track/heaptrack.sh.cmake

```diff
@@ -96,11 +96,15 @@ while true; do
             exit 0
             ;;
         "-o" | "--output-file")
-            output=$2
-            if [ -z "$output" ]; then
+            if [ -z "$2" ]; then
                 echo "Missing output argument."
                 exit 1
             fi
+            output=$(readlink -f $2)
+            if [ -d "$output" ]; then
```

Comment:
will this catch the case where `$2` points to an invalid path (e.g. a subdir that doesn't exist) and then `output` 
becomes empty? Or do we need to check for ` || [ -z "$output" ]` too?

Reply:
It falls back to the default output path.